benchmarking/config.ini

```diff
@@ -0,0 +1,15 @@
+[User Constants]
```

Comment:
Just `User` should be fine. The term `Constants` is redundant here. It is after all a config file.

Reply:
Yes, changed.